.github/workflows/ci.yml

```diff
@@ -128,11 +128,53 @@ jobs:
       - name: rustdoc
         run: RUSTDOCFLAGS="-Dwarnings" ./miri cargo doc --document-private-items
 
+  sysroots:
```

Comment:
Would it make sense to use a separate workflow (in a separate yml file) for this? It doesn't seem to share anything with the main workflow except the time of the cronjob.

Reply:
I tested this PR in my fork and ironed out a few issues. Let me see if I can cut-and-paste without breaking this...

Reply:
I don't see any evidence that GitHub has picked up the new workflow. But it seems to have been picked up in my own repo? https://github.com/saethlin/miri/pull/1

Reply:
It may be the case that it only gets picked up once it exists in master.

Reply:
That's what I thought too, but in my fork the workflow doesn't exist on master.

Reply:
I believe for your own repo it will apply every workflow found on a local branch, but for a PR it will only apply those found on the target branch.

Reply:
If you click the link you'll see the sysroots job running in a PR, no?

Reply:
You are the owner of that repo, so your PRs are not regular PRs.

Reply:
I did not know that. That is very annoying.